db/migrations/20240411212105_add_ext_user_table.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex) {
+  return knex.schema.createTable("external_user", function (table) {
+    table.string("external_id").notNullable().primary().unique();
```

Comment:
Should we put a length constraint on this, too?
```suggestion
    table.string("external_id", 64).notNullable().primary().unique();
```